### ISSUE TITLE
Is it possible to compress on the fly and extract in the client side player, the reason for asking this is pricing will be more if we opt for RTMP to RTMP in AWS

### ISSUE THREAD

**Issue by sriman** [arut/nginx-rtmp-module#1251]

**Comment by misiek08** [comment]
tl;dr Don't use AWS for live streaming.

If you have some data processing services you can host them on AWS, but
pricing for bandwidth in AWS is one of the highest in the market. Most
companies host static and/or high-bandwidth content outside "cloud" to use
right tool for the job.

wt., 24.04.2018, 07:10 użytkownik sriman <notifications@github.com> napisał:

**Comment by mrfrieza** [comment]
Why don't you host it yourself ? Just gotta open the hls port on your internet box settings... then it'll be playable from anywhere with the right address..

**Comment by sriman** [comment]
Yes we need to host on our own, [Packet](https://www.packet.com/) is good one to host with less cost and using the BGP routing [https://metallb.universe.tf/concepts/bgp/](url)
